### DISCUSSION TITLE
Gollum website

### DISCUSSION THREAD

**Discussion by Alxira5** [gollum/gollum#2039] [Ideas]
Hello developers, I wanted to tell you a suggestion regarding the project and I think that because it does not have a website like other projects?, can be a website with screenshots and the history and history of the project. The website can be built with [Jekyll](https://jekyllrb.com) and hosted on [GitHub Pages](https://pages.github.com/).

It is a very simple suggestion and I think it would do you good in the exploration and discovery of the project.

**Comment by benjaminwil** [comment]
Thanks for submitting this idea. It's a good one.

I can't speak for the other maintainers, really, but I currently don't have the capacity to do this work any time soon.

The history of this project is incredibly interesting, so if someone were start compiling that information in [the Gollum wiki](https://github.com/gollum/gollum/wiki) that would be a great contribution for the future project website. 👍

**Reply by Alxira5** [reply]
I can create a Gollum website with some interesting information about the project and a script to export the wiki statically. There is a tool called [gollum-site](https://github.com/dreverri/gollum-site), but it is a bit outdated and I plan to turn it into a plugin for [Jekyll](https://jekyllrb.com/docs/plugins/).

Because of this, I suppose a CI should be used for its launch.
